src/main/java/minicraft/screen/PlayerInvDisplay.java

```diff
@@ -7,54 +7,77 @@
 import minicraft.entity.mob.Player;
 import minicraft.gfx.Color;
 import minicraft.gfx.Font;
+import minicraft.gfx.Point;
 import minicraft.gfx.Screen;
 import minicraft.item.Inventory;
 import minicraft.item.Item;
 import minicraft.item.Items;
 import minicraft.item.StackableItem;
+import minicraft.screen.entry.StringEntry;
 
 public class PlayerInvDisplay extends Display {
 
 	private static final int padding = 10;
 
 	private final Player player;
 
+	private String itemDescription = "";
+	private Menu.Builder descriptionMenuBuilder;
```

Comment:
You don't need this.

Reply:
Why we don't need it? It is a dynamic menu size.

Reply:
Just don't keep the builder stored.

Reply:
The builder is always stored every time the frame sizes are modified. It also appears in other places as commonly.